src/main/java/server/HTTPServer.java

```diff
@@ -1,10 +1,47 @@
 package server;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 public class HTTPServer implements Application {
+    ArrayList<String> request = new ArrayList<>();
+
+    List<String> resources = List.of("/");
+
+    String requestedResource;
+
     @Override
     public void call(ReadableWriteable readerWriter) throws IOException {
-        readerWriter.writeLine("HTTP/1.1 200 OK\r\nContent-Length:0\r\n");
+        readRequest(readerWriter);
+        parseRequest();
+        writeResponse(readerWriter);
+        resetRequest();
```

Comment:
**question**: Why do we need to reset? How can we refactor to eliminate that need?

Resetting an object's state is a bit of a code smell for me. In my experience, there is typically another way to do it that eliminates the need entirely.